cmd/create/cluster/cmd_test.go

```diff
@@ -474,7 +474,7 @@ var _ = Describe("getInitialValidSubnets()", func() {
 	})
 
 	It("OK: get valid subnets", func() {
-		validSubnets, err := getInitialValidSubnets(mockClient, ids, r.Reporter)
+		validSubnets, err := getInitialValidSubnets(false, mockClient, ids, r.Reporter)
```

Comment:
@davidleerh Would it be possible to add a test for setting the `hostedCp` variable to `true`.

Reply:
my mistake here. 
After review.
It seems it was already implemented on the "rosa create cluster" side of things via 
https://github.com/openshift/rosa/pull/1790

Will remove the change on "rosa create cluster" -> "getInitialValidSubnets".